### PR TITLE
Updates to Rails 6 and Spree 4

### PR DESCRIPTION
**Pull request by PSzyszkaSpark**
Fixes https://spark-solutions.atlassian.net/wiki/spaces/SOS/pages/848986120/Updating+Extensions+to+Rails+6+and+Spree+4

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,5 +2,6 @@ source 'https://rubygems.org'
 
 gem 'rails-controller-testing'
 gem 'spree', github: 'spree/spree', branch: 'master'
+gem 'ckeditor', github: 'galetahub/ckeditor', branch: 'master', ref: '4385f23b90f84c25f7b912bd75c233c8f27b5f58'
```
Comment (damianlegawiec) [inline Gemfile]: @PSzyszkaSpark we don't need to use the master branch,. 5.0.0 is already released

File: Appraisals [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
+appraise 'spree-3-7' do
+  gem 'spree', '~> 3.7.0'
   gem 'rails-controller-testing'
+  gem 'sass-rails'
 end
 
-appraise 'spree-3-6' do
-  gem 'spree', '~> 3.6.0'
+appraise 'spree-4-0' do
+  gem 'spree', '~> 4.0.0.rc2'
```
Comment (damianlegawiec) [inline Appraisals]:
```suggestion
  gem 'spree', '~> 4.0.0'
```

File: Gemfile [hunk 3]
```diff
@@ -2,5 +2,6 @@ source 'https://rubygems.org'
 
 gem 'rails-controller-testing'
 gem 'spree', github: 'spree/spree', branch: 'master'
+gem 'ckeditor', github: 'galetahub/ckeditor'
```
Comment (damianlegawiec) [inline Gemfile]:
```suggestion
```

let's use rubygem version please

File: config/initializers/ckeditor.rb [hunk 4]
```diff
@@ -16,6 +16,14 @@
...
-    config.authorize_with :cancan, Spree::Ability
+    config.authorize_with :cancancan, Spree::Ability
+
+    # Asset model classes
+    config.picture_model { Ckeditor::Picture }
+    config.attachment_file_model { Ckeditor::AttachmentFile }
+
+    # //cdn.ckeditor.com/<version.number>/<distribution>/ckeditor.js | distributions
+    # available here: https://ckeditor.com/cke4/presets-all
+    config.cdn_url = '//cdn.ckeditor.com/4.11.3/standard/ckeditor.js'
```
Comment (damianlegawiec) [inline config/initializers/ckeditor.rb]: @PSzyszkaSpark shouldn't be 5.0.0 also?

File: gemfiles/spree_4_0.gemfile [hunk 5]
```diff
@@ -3,6 +3,6 @@
 source "https://rubygems.org"
 
 gem "rails-controller-testing"
-gem "spree", "~> 3.2.0"
+gem "spree", "~> 4.0.0.rc2"
```
Comment (damianlegawiec) [inline gemfiles/spree_4_0.gemfile]:
```suggestion
gem "spree", "~> 4.0.0"
```

File: config/initializers/ckeditor.rb (hunk 4 above)
Comment (PSzyszkaSpark) [reply config/initializers/ckeditor.rb]: If I change it to 5.0.0 here the specs starts failing.

**Pull request merged**